src/components/OptionModal.tsx

```diff
@@ -11,6 +11,12 @@ import { XorMaskType, xorMaskTypes } from '../xorMask';
 
 import './OptionModal.css';
 
+export enum showCenterLine {
```

Comment:
```suggestion
export enum ShowCenterLine {
```
大文字にすべきでしたね。昔の不注意ですみません。